src/view/pages/__snapshots__/gettingStarted.spec.tsx.snap

```diff
@@ -116,73 +208,142 @@ exports[`GettingStartedPage component  should render correctly 1`] = `
   </div>
   <div
     className="inv"
-    id="CPX"
+    id="CLUE"
   >
     <h2>
-       Tutorial for CPX 
+       Tutorial for CLUE 
     </h2>
     <h3>
-      1. Import the micro:bit library to use it (This is required)
+      1. Import the the main CLUE library (This is required)
     </h3>
     <span
       className="codeBox"
     >
       <pre>
-        from adafruit_circuitplayground import cp
+        from adafruit_clue import clue
       </pre>
     </span>
     <h3>
-       2. Turn on the little red LED
+      2. Display text on the CLUE and change the text when a button is pressed
     </h3>
     <span
       className="codeBox"
     >
+      <pre>
+        clue_data = clue.simple_text_display(title="CLUE!", text_scale=2)
+      </pre>
       <pre>
         while True:
       </pre>
       <pre>
-         cp.red_led = True
+            clue_data[1].text = "Hello World!"
+      </pre>
+      <pre>
+            clue_data[3].text = "Temperature: 
+        {}
+        ".format(clue.temperature)
+      </pre>
+      <pre>
+            if clue.button_a:
+      </pre>
+      <pre>
+                clue_data[5].text = "A is pressed!"
+      </pre>
+      <pre>
+            else:
+      </pre>
+      <pre>
+                clue_data[5].text = "A is not pressed!"
+      </pre>
+      <pre>
+            clue_data.show()
       </pre>
     </span>
     <h3>
-       3. Turn up red LED when button A is clicked
+       3. Create a slide show on the CLUE
     </h3>
+    <p>
+      There must be some .bmp photos in the same directory as the code file.
+    </p>
     <span
       className="codeBox"
     >
       <pre>
-        while True:
+        import board
       </pre>
       <pre>
-         if cp.button_a:
+        from adafruit_slideshow import SlideShow
       </pre>
       <pre>
-         cp.red_led = True
+         
+      </pre>
+      <pre>
+        slideshow = SlideShow(board.DISPLAY, auto_advance=True)
```

Comment:
Perhaps it'd be cool to add arguments for fade_effect and dwell so that users know that they can manipulate those